torch/csrc/lazy/core/lazy_graph_executor.h

```diff
@@ -283,15 +274,25 @@ class TORCH_API LazyGraphExecutor {
     std::vector<BackendDataPtr> tensors_data;
   };
 
+  void ResetTrimCounter() const;
+
+  // Waits for this SyncTensorCollection's device barrier and acuire the lock.
+  virtual void TensorCollectionBarrier(SyncTensorCollection* coll);
+
+ private:
```

Comment:
Should this be `protected` now? I'm not sure I understand why it was moved otherwise

Reply:
Because we have `protected` on the top already. I didn't move the `private` just diff being diff...

Reply:
wait didn't you move a private?  but i think there's another private above anyway so nothing changed? (above TensorHasher, L237)

Reply:
I moved `TensorCollectionBarrier` and `ResetTrimCounter` to the protected space.